bindata/network/multus-admission-controller/001-service.yaml

```diff
@@ -13,7 +13,10 @@ metadata:
 {{- if .HyperShiftEnabled}}
     network.operator.openshift.io/cluster-name: {{.ManagementClusterName}}
 {{- end }}
+# Any new HCP deployment will automatically reconcile this secret in the HCP without the service-ca operator
+{{- if not .HyperShiftEnabled}}
```

Comment:
To work in a hypershift env that doesn't reconcile the secrets you also need the additional CAConfigMap check here.